docs/source/rest_api.rst

```diff
@@ -49,7 +46,7 @@ Request Structure
 +------------+------------+-------------------------+
 | Field Name |    Type    |       Description       |
 +============+============+=========================+
-| name       | ``STRING`` |                         |
+| name       | ``STRING`` | Experiment name.        |
 |            |            | This field is required. |
 |            |            |                         |
 +------------+------------+-------------------------+
```

Comment:
Were these auto-generated from a protobuf or written by hand? Would be better to change the protobuf if that's where they're from.

Reply:
The problem is that the generator has bugs. For the rest of the Databricks API docs we initially generate them from protobuf, but then we maintain them by hand.

Reply:
I see. @mparkhe can you comment on this too? I'm okay switching to this but we need to make sure we don't regenerate them and lose the changes here.

Reply:
+1 to that. If there is a way to backfill the hand generated changes to PBs (after the said bugs are removed) that would be awesome. Esp., as we add more functionality and potentially more API changes to PBs, maintaining it by hand is painful.